### PR TITLE
[stable/elastalert] Updated image for Elasticsearch 6 support, and other minor chart adjustments

### PR DESCRIPTION
**Pull request by jertel**
Updated image for Elasticsearch 6 support, removed supervisord since k8s deployments provide that capability already, and added apiVersion to support helm upgrades

*****************************************************************************************

Thank you for contributing to kubernetes/charts. Before you submit this PR we'd like to
make sure you are aware of our technical requirements and best practices:

* https://github.com/kubernetes/charts/blob/master/CONTRIBUTING.md#technical-requirements
* https://github.com/kubernetes/helm/tree/master/docs/chart_best_practices

For a quick overview across what we will look at reviewing your PR, please read
our review guidelines:

* https://github.com/kubernetes/charts/blob/master/REVIEW_GUIDELINES.md

Following our best practices right from the start will accelerate the review process and
help get your PR merged quicker.

When updates to your PR are requested, please add new commits and do not squash the 
history. This will make it easier to identify new changes. The PR will be squashed 
anyways when it is merged. Thanks.

*****************************************************************************************

### CODE REVIEW

**Comment by unguiculus** [comment]
cc @pickledrick

**Comment by karlskewes** [comment]
Thank you for this. Have you published your dockerfile and/or image somewhere? I can't see it on docker hub. "jertel/elastalert-docker". 
Or are you using this Ivan Krizsan's dockerfile or a version of it? https://hub.docker.com/r/ivankrizsan/elastalert/
Keen to test.

**Comment by jertel** [comment]
Hello, yes I have the github repo containing the Dockerfile linked to the Docker hub build. It's on the right side of the screen when viewing the Docker hub page. Here's the direct link: https://github.com/jertel/elastalert-docker. This Dockerfile is very simple, stripped down to basics. I'm using this image today against an Elasticsearch 6.0 installation and it works well. Thanks for taking the time to review.

**Comment by viglesiasce** [comment]
@jertel Please add your Dockerfile repo to the sources list

**Comment by jertel** [comment]
Which sources list?

**Comment by unguiculus** [comment]
@jertel Update the sources list in `Chart.yaml`.

**Comment by jertel** [comment]
I added the new repo to source list and removed the obsolete repo. Thanks.

**Comment by jertel** [comment]
Anything I need to do on this? The test appears to have failed because the pod shuts down due to the inability to connect to an elasticsearch server (which makes sense since there is no such server included in this chart). I'm not waiting on the pull-req to be merged personally but if there's something I can do to help get it closed and off your PR list, let me know.

**Comment by unguiculus** [comment]
CI has to be green. Otherwise the bot won't merge. You'll may have to conditionally create the deployment.

File: stable/elastalert/Chart.yaml [hunk 1]
```diff
@@ -1,12 +1,13 @@
...
+appVersion: 0.1.29
 icon: https://static-www.elastic.co/assets/blteb1c97719574938d/logo-elastic-elasticsearch-lt.svg
 sources:
-- https://github.com/krizsan/elastalert-docker
+- https://github.com/jertel/elastalert-docker
 - https://github.com/Yelp/elastalert
 maintainers:
   - name: pickledrick
     email: 9246508+pickledrick@users.noreply.github.com
+  - name: Jason Ertel
```
Comment (unguiculus) [inline stable/elastalert/Chart.yaml]: Use Github username and add e-mail.

**Review by unguiculus — CHANGES_REQUESTED** [review]

**Review by unguiculus — APPROVED** [review]

**Pull request merged**